server/src/repositories/Equipment/ReservationRepository.ts

```diff
@@ -6,10 +6,14 @@ import { reservationsToDomain, singleReservationToDomain } from "../../mappers/e
 export interface IReservationRepository {
   getReservationById(id: number | string): Promise<Reservation | null>;
   getReservations(): Promise<Reservation[]>;
-  addReservation(reservation: ReservationInput): Promise<Reservation | null>;
+  createReservation(reservation: ReservationInput): Promise<Reservation | null>;
+  assignLabbieToReservation(resId: number, labbieId: number): Promise<Reservation | null>;
+  removeLabbieFromReservation(resId: number): Promise<Reservation | null>;
+  addComment(resId: number, authorId: number, commentText: string): Promise<string | null>;
```

Comment:
I don't think we need `authorID` since we can get that from the context.

Reply:
@MatthewGalan https://github.com/rit-construct-makerspace/makerspace/pull/125/files you added creator in the type. Should we get rid of this? I was following this schema